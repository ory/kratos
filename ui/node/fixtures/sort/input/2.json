[
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "method",
      "type": "submit",
      "value": "password",
      "disabled": false
    },
    "messages": null
  },
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "password",
      "type": "password",
      "minlength": 8,
      "maxlength": 1024,
      "required": true,
      "disabled": false
    },
    "messages": null
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "method",
      "type": "submit",
      "value": "oidc",
      "disabled": false
    },
    "messages": null
  },
  {
    "type": "input",
    "group": "default",
    "attributes": {
      "name": "csrf_token",
      "type": "hidden",
      "value": "qomtJu6UOzh5Gqkq3MEwbbJuNgJEfDg9gEowfMH0OhjfAIcWbc0PrbvMS6LL7m8ptuuuQE+oUDoXUQOy/kRccw==",
      "required": true,
      "disabled": false
    },
    "messages": null
  },
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "traits.email",
      "type": "email",
      "disabled": false
    },
    "messages": null
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "oidc.provider",
      "type": "submit",
      "value": "github",
      "disabled": false
    },
    "messages": null
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "oidc.provider",
      "type": "submit",
      "value": "google",
      "disabled": false
    },
    "messages": null
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "oidc.provider",
      "type": "submit",
      "value": "hydra",
      "disabled": false
    },
    "messages": null
  },
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "traits.abcd",
      "type": "input",
      "disabled": false
    },
    "messages": null
  }
]
