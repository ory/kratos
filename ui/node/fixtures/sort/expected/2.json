[
  {
    "type": "input",
    "group": "default",
    "attributes": {
      "name": "csrf_token",
      "type": "hidden",
      "value": "qomtJu6UOzh5Gqkq3MEwbbJuNgJEfDg9gEowfMH0OhjfAIcWbc0PrbvMS6LL7m8ptuuuQE+oUDoXUQOy/kRccw==",
      "required": true,
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "oidc.provider",
      "type": "submit",
      "value": "github",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "oidc.provider",
      "type": "submit",
      "value": "google",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "oidc.provider",
      "type": "submit",
      "value": "hydra",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "method",
      "type": "submit",
      "value": "oidc",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "traits.abcd",
      "type": "input",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "password",
      "type": "password",
      "required": true,
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "traits.email",
      "type": "email",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "method",
      "type": "submit",
      "value": "password",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  }
]
